apps/XiaoMiTool/install

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+install_packages openjfx adb android-tools-fastboot openjdk-11-jdk || exit 1
+
+git_clone https://github.com/francescotescari/XiaoMiToolV2 -b linux || exit 1
+cd XiaoMiToolV2
+echo "Building XiaoMi Tool..."
+case "$arch" in
+    "32") JAVA_HOME=/usr/lib/jvm/java-11-openjdk-armhf ./gradlew build || error "Failed to build XiaoMiToolV2!" ;;
+    "64") JAVA_HOME=/usr/lib/jvm/java-11-openjdk-arm64 ./gradlew build || error "Failed to build XiaoMiToolV2!" ;;
+    *) error "Failed to detect OS CPU architecture! Something is very wrong." ;;
+esac
+echo "Creating menu shortcut..."
+echo "[Desktop Entry]
+Name=XiaoMi Tool
+Comment=XiaoMiTool is the all-in-one tool to make modding easy for everyone with a Xiaomi smartphone.
+Exec=gksudo -k -u root JAVA_HOME=$JAVA_HOME $HOME/XiaoMiToolV2/gradlew run
```

Comment:
```suggestion
Exec=JAVA_HOME=$JAVA_HOME $HOME/XiaoMiToolV2/gradlew run
```

gksudo is not available in Raspbian. Is it really needed?

Reply:
neither is `gksu`..

Reply:
As far as I can tell, it doesn't need sudo to run.